Compatibility issues with Photon

```
[ERROR] lua/autorun/photon/sh_photon_vehicles.lua:4: Tried to use a NULL entity!
  1. GetModel - [C]:-1
   2. RecoverVehicleTable - lua/autorun/photon/sh_photon_vehicles.lua:4
    3. unknown - lua/autorun/photon/sh_photon_vehicles.lua:35

Timer Failed! [243-PHOTON-57.909091949463][@lua/autorun/photon/sh_photon_vehicles.lua (line 28)]
```

Seems to kick you out when switching seats and sometimes causes the above server error when an aircraft is spawned. Unsure whether WAC or Photon is at fault currently.

--- a/WAC Base/lua/entities/wac_seat_connector/init.lua
+++ b/WAC Base/lua/entities/wac_seat_connector/init.lua
@@ -76,6 +76,9 @@ function ENT:switchSeat(p, int)
 	local oldang = p:GetAimVector():Angle()
 	oldang.y = oldang.y+90
 	p:ExitVehicle()
+	if VCMod1 then
+		p.VC_CanEnterTime = CurTime()
+	end
 	p:EnterVehicle(self.seats[int])
 	--p:SnapEyeAngles(self.seats[int]:GetAngles())
 	self:updateSeats()

